client/OPUNetGameSelectWnd.cpp

```diff
@@ -149,6 +145,15 @@ void OPUNetGameSelectWnd::OnInit()
 	timer = SetTimer(this->hWnd, 0, timerInterval, nullptr);
 }
 
+void OPUNetGameSelectWnd::InitializePlayerNameComboBox()
+{
+	char buffer[MaxServerAddressLen];
+
+	// Set the default player name
+	GetPlayerName(buffer, false);
+	SetDlgItemText(this->hWnd, IDC_PlayerName, buffer);
```

Comment:
We could potentially use a pre-sized `std::string` here. Though the raw array is closer in style to the C based API being used here.